bash/global_variables.bash

```diff
@@ -154,10 +160,17 @@ function Define_Further_Global_Variables()
         [Sampler]=''
         [Afterburner]=''
     )
+    declare -gA HYBRID_module=(
+        [Sampler]='SMASH'
```

Comment:
Shouldn't this be `=${HYBRID_default_sampler_module}`? Apart from here, where is the variable `HYBRID_default_sampler_module` used?

Reply:
True, this was leftover from refactoring.